app/src/main/AndroidManifest.xml

```diff
@@ -128,6 +133,9 @@
             <intent-filter>
                 <action android:name="android.intent.action.SEARCH" />
             </intent-filter>
+            <meta-data
+                android:name="android.app.searchable"
+                android:resource="@xml/searchable" />
```

Comment:
It might be better to open the MainActivity with the search intent instead of the SearchActivity

Reply:
this would also move the search intent filter